src/System.Diagnostics.Process/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netstandard-Unix;
+      uap-Windows_NT;
+      uapaot-Windows_NT;
```

Comment:
If we need the uap specific configuration then we will also need a uapaot specific configuration if we want the tests to run correctly.